setup.py

```diff
@@ -17,7 +17,7 @@
 
 setup(
     name='sctools',
-    version='0.1.0',
+    version='0.1.4',
     description='Utilities for large-scale distributed single cell data processing',
     url='https://github.com/humancellatlas/scsequtil.git',
```

Comment:
Should this be `https://github.com/HumanCellAtlas/sctools.git` or you would change the repo name to `scsequtil` later?